client/src/main/java/io/pravega/client/stream/ReaderGroupConfig.java

```diff
@@ -49,11 +50,37 @@
 
     private final int maxOutstandingCheckpointRequest;
 
+    private final StreamDataRetention retentionConfig;
+
+    /**
+     * The values of this enum indicate if a ReaderGroup chooses Consumption based data retention for Streams it reads from.
+     * For Consumption based retention to work, in addition to setting the appropriate value here,
+     * the Streams' Retention Policy must be set to 'CONSUMPTION'.
+     *
+     * NO_IMPACT - Read Positions of Readers do not impact Stream truncation.
+     * CONSUMPTION_BASED_USER_STREAMCUT - ConsumptionBasedRetention, user provides the StreamCut for truncation.
```

Comment:
As this setting only affects consumption based retention, having it in the name is somewhat redundant.

Reply:
Ok. changed this.